### ISSUE TITLE
Support InTune deployments for the same app to different (InTune) distribution groups

### ISSUE THREAD

**Issue by pcampbell83** [microsoft/appcenter#1454]
### Problem
TLDR; we wish to allow a subset of the user-base access to our app earlier than all other users in production, via the AppCenter Intune distribution method.

If you setup multiple AppCenter/Intune store connections _for the same app_, distribution to these store configurations results in the target distribution group being added into the release in Intune.

The net result is that over time the distribution you select in AppCenter no longer only targets the configured group, but any group who was ever distributed to for that app.

### Context
We have our user-base split into two groups: all production users, and a pilot group that is ~20% of the  total user-base. When we release a new version of our app, we roll out to the pilot user-base a few weeks _before_ all other users to minimise disruption in the event of major problems. Currently, if we distribute to pilot via AppCenter/InTune store connection, the resultant InTune distribution is appended to when viewed in the Intune dashboard, rather than replacing, meaning over time you end up distributing to more than the store configured group. 

### Alternatives we considered
We considered a completely seperate app that is distributed to the pilot users alone, but this would require a different pipeline, tracking and release management in addition to what we already have.

Our pilot users are expected to be working with the latest **prod** release and we simply wish to allow a subset of the user-base access to it earlier. Call it early-access, if you will. It it not a test, beta, alpha or UAT version. It is a prod version given to a select group prior to large-scale deployment.

**Comment by MichaelSafro** [comment]
Wanted to add that we encountered a somewhat similar situation and I find the requested method to be useful.

**Comment by MichaelSafro** [comment]
@Oddj0b - Hi Villars, thanks for picking this up! I wonder if you think this request, is doable, and in case it is, how long do you think it would take to make it working?

**Comment by Oddj0b** [comment]
Hey @MichaelSafro, we normally use feature requests as an indicator of how many people we will help if we implement a request. And to help the most we're dedicating time to pick the top requested features. My hope is that this request gets more 👍 so that we can start working on it.
